lib/views/main_screen.dart

```diff
@@ -187,30 +190,51 @@ class _MainScreenState extends State<MainScreen> {
             "class": const ChatPage(
               key: Key('Chats'),
             ),
+          },
+          "Donation": {
+            "class": const ChangeThemeTile(),
```

Comment:
We don't need this for now.

Reply:
The problem is with the plugins being added dynamically. If I remove this line, then the execution will never reach to the code with loads plugins (as in the below image). I added this just for the sake of testing purposes. If you have any better ideas on how to test that, please let me know.

![image](https://user-images.githubusercontent.com/62198564/217895207-e043b620-c01d-4e8f-b5a8-aee07d774b44.png)

Reply:
@palisadoes Your thoughts on this ?

Reply:
Keep the line and create an issue on how to handle plugins better.

Reply:
Yes